specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -6019,6 +6067,96 @@
       },
       "discriminator": "instanceType"
     },
+    "HealthError": {
+      "description": "Health Error",
+      "type": "object",
+      "properties": {
+        "errorSource": {
+          "description": "Source of error.",
+          "type": "string"
+        },
+        "errorType": {
+          "description": "Type of error.",
+          "type": "string"
+        },
+        "errorLevel": {
```

Comment:
These are pre approved classes/object model and any change now will be a breaking change.